where-to-travel/src/main/webapp/script.js

```diff
@@ -106,23 +117,55 @@ function showInfoModal() {
  *
  * @param {boolean} useAddress Flag indicating whether to get location from browser or address
  */
-function getHomeLocation(useAddress) {
-  let locationFunction = () => getLocationFromBrowser();
+function getHomeLocation(useAddress) { 
+  const addressInput = document.getElementById('addressInput');
 
+  let locationFunction = () => getLocationFromBrowser();
```

Comment:
Place this in an else function instead? To prevent this being viewed as default.

Reply:
fixed!